### DISCUSSION TITLE
Not authorized to execute command on connection deployed mongo database

### DISCUSSION THREAD

**Discussion by prajwal497** [feathersjs/feathers#2824] [General]
### Steps to reproduce

Create new app using feathers dove pre released cli.
feathers generate app
Select Mongo Db as adapter

### Expected behavior
Db calls should work through generated user services.

### Actual behavior
Getting error on every service call:
not authorized on {dbname}?retryWrites=true&w=majority to execute command { insert: \"user\", documents: [ { email: \"user@email.com\", password: \"$2a$10$nr.fA8Rg2ygFoT/pEMBct.OGmYydNc8zTbJjPj31VSV7aG21/CZJi\", _id: ObjectId('63456b4a678cad735ee797c6') } ]

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**Feathers Version**: ^5.0.0-pre.30

**NodeJS version**:  16.17.0

**Comment by daffl** [comment]
My test with a local MongoDB worked so my guess is that you need to update the MongoDB connection string to what the running MongoDB requires. I believe e.g. MongoDB Atlas shows the correct one in their Dashboard.
